### PR TITLE
Environment Variable fixes 

### PR DESCRIPTION
**Pull request by chuck-alt-delete**
## What type of PR is this? (check all applicable)

- [x] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [x] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Add debug messages for environment variables and show exports in .env.example so users can more easily run datagen from a different directory from the .env file

## Added to documentation?

- [x] 📜 readme
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-KAFKA_BROKERS=
+export KAFKA_BROKERS=
 
 # Kafka SASL Authentication
-SASL_USERNAME=
-SASL_PASSWORD=
-SASL_MECHANISM=
+export SASL_USERNAME=
+export SASL_PASSWORD=
+export SASL_MECHANISM=
```
Comment (bobbyiliev) [inline .env.example:8]: I like the idea of having `export` in the README.md file, but not sure if this is a good practice for the `.env.example`.

In general, with `.env` file in Node.js and other frameworks, you should not use the export keyword within the file itself. 

Not sure if the `dotenv` library, expects the `export` keyword on each line. It usually is just a key-value pair.

Comment (chuck-alt-delete) [reply .env.example:8]: Dotenv is totally cool with export. It makes it so you can `source /path/to/.env; datagen <blah>`

Comment (bobbyiliev) [reply .env.example:8]: oh ok, in this case we should be good to go!

**Review by bobbyiliev — APPROVED** [review]
Once we merge this we can go ahead and cut a new release to include the Postgres destination! 🎉

**Pull request merged**